lib/meetings.js

```diff
@@ -70,19 +76,68 @@ const shouldCreateNextMeetingIssue = module.exports.shouldCreateNextMeetingIssue
   return issue
 }
 
-const getNextScheduledMeeting = module.exports.getNextScheduledMeeting = function (schedules = [], now = DateTime.utc()) {
+const INVALID_MESSAGE = 'invalid schedule format'
+const GUIDANCE_MESSAGE = 'use either a timezone identifier (e.g., 2020-04-02T17:00:00[America/Chicago]) or UTC (e.g., 2020-04-02T17:00:00Z)'
+
+function parseScheduleString (scheduleStr) {
+  const lastSlashIndex = scheduleStr.lastIndexOf('/')
+  const startStr = scheduleStr.substring(0, lastSlashIndex)
+  const durationStr = scheduleStr.substring(lastSlashIndex + 1)
+
+  return {
+    startStr,
+    duration: Temporal.Duration.from(durationStr)
+  }
+}
+
+function parseDateAndTimezone (startStr) {
+  const hasOffset = /[+-]\d{2}:?\d{2}/.test(startStr)
+  const hasTimezone = /\[[^\]]+\]/.test(startStr)
+  const hasUTC = /Z$/.test(startStr)
+  let instant, timezone
```

Comment:
```suggestion
  let instant
  let timezone
```

Sorry for the nit pick, but not only did I think the linter would have complained on this, I will personally. I have two unreasonable opinions as far as formatting goes:

1. single variable declarations only: Except when de-structuring, this pattern ends with sloppy code and bad diffs later for zero benefit.
2. Always use `{}` and `return` in arrow functions: similarly ends with hard to read sloppy code and bad diffs for no benefit.

Terseness when typing is *never* more important than the time spent later reading the code. I usually wouldn't make these types of comments as they are silly and nitpicking, but I know you and I have some fun discussions when it comes to silly pedantic opinions like this lol, so figured I would. Not blocking, to be clear.